lyncs_quda/dirac.py

```diff
@@ -237,20 +235,23 @@ def action(self, phi, **params):
                     "computeTrLog should be set True in the preconditioned case"
                 )
 
+        out = 0
+        if not self.full and "CLOVER" in self.type:
+            self.quda_dirac
+            self.clover.inverse_field
```

Comment:
The calculation of the inverse_field isn't more appropriate in `trLog`?
And why do you need to call `self.quda_dirac`?